src/app/components/trade/LPDescription.tsx

```diff
@@ -111,18 +87,19 @@ export default function LPDescription({ baseSuplying, quoteSupplying }: ILPDescr
               Your supply
             </Typography>
             <Typography textAlign="center" variant="p">
-              {selectedPair.id !== BalancedJs.utils.POOL_IDS.sICXICX ? (
+              {!pair?.queueRate ? (
```

Comment:
Using ID looks more explicit. We will keep the sICXICX pool ID constant for sure in the future, but the queueRate can be renamed or changed.

Reply:
I got you